images-list

```diff
@@ -609,6 +609,7 @@ rancher/metrics-server rancher/mirrored-metrics-server v0.3.4
 rancher/metrics-server rancher/mirrored-metrics-server v0.3.6
 rancher/nginx-ingress-controller-defaultbackend rancher/mirrored-nginx-ingress-controller-defaultbackend 1.5-rancher1
 rpardini/docker-registry-proxy rancher/rpardini-docker-registry-proxy 0.6.1
+sigwindowstools/k8s-gmsa-webhook rancher/mirrored-sigwindowstools-k8s-gmsa-webhook v0.3.0
```

Comment:
Waiting on: https://hub.docker.com/r/rancher/mirrored-sigwindowstools-k8s-gmsa-webhook

Reply:
This has been created.